ml-agents-envs/mlagents/envs/mock_communicator.py

```diff
@@ -8,9 +8,10 @@
 from mlagents.envs.communicator_objects.unity_input_pb2 import UnityInputProto
 from mlagents.envs.communicator_objects.unity_output_pb2 import UnityOutputProto
 from mlagents.envs.communicator_objects.agent_info_pb2 import AgentInfoProto
-from mlagents.envs.communicator_objects.compressed_observation_pb2 import (
-    CompressedObservationProto,
-    CompressionTypeProto,
+from mlagents.envs.communicator_objects.observation_pb2 import (
+    ObservationProto,
+    NONE as COMPRESSION_TYPE_NONE,
```

Comment:
This is a bit awkward, but CompressionTypeProto.NONE doesn't work on protobuf<3.8